R/try.R

```diff
@@ -16,21 +16,19 @@
 
     call <- sys.call(-1)
     env <- parent.frame()
-    try <- substitute(try)
-    fallback <- substitute(except)
 
     tryCatch(
-        eval(try, envir = env),
+        try,
         error = function(e) {
             log_level(
                 WARN,
                 paste(
-                    'Running', shQuote(deparse(fallback)), 'as',
-                    shQuote(deparse(try)), 'failed:',
+                    'Running', shQuote(deparse(substitute(except))), 'as',
+                    shQuote(deparse(substitute(try))), 'failed:',
```

Comment:
I think the `substitute` calls need to remain outside of the `error` fn of `tryCatch`?

Reply:
See e.g. 

```r
> f <- function() invisible()
> f(42) %except% 42
WARN [2024-08-07 09:16:33] Running 'except' as 'try' failed: 'unused argument (42)'
[1] 42
```

Reply:
Ok, I'll add test.